src/src/com/microsoft/aad/adal/SerializedTokenCacheItem.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.aad.adal;
+
+import com.google.gson.annotations.SerializedName;
+
+final class SerializedTokenCacheItem {
+
+    private static final String TAG = SerializedTokenCacheItem.class.getSimpleName();
+
+    @SerializedName("authority")
+    private final String mAuthority;
+
+    @SerializedName("refreshToken")
+    private final String mRefreshToken;
+
+    @SerializedName("idToken")
+    private final String mIdToken;
+
+    @SerializedName("familyClientId")
+    private final String mFamilyClientId;
+
+    SerializedTokenCacheItem(final TokenCacheItem tokenCacheItem) {
+        this.mAuthority = tokenCacheItem.getAuthority();
+        this.mRefreshToken = tokenCacheItem.getRefreshToken();
+        this.mIdToken = tokenCacheItem.getRawIdToken();
```

Comment:
May I ask what is more lightweight in SerializedTokenCacheItem  in compare to TokenCacheItem?
It just doesn't contain access token and expiration date. Which family token shouldn't contain anyway...